js/private/js_image_layer.bzl

```diff
@@ -18,9 +18,16 @@ load("@bazel_skylib//lib:paths.bzl", "paths")
 
 _DOC = """Create container image layers from js_binary targets.
 
-js_image_layer supports transitioning to specific platform for cross-compiling.
+By design, js_image_layer doesn't have any preference over which rule assembles the container image. 
+This means the downstream rule (`oci_image`, or `container_image` in this case) must set a proper `workdir` and `cmd` to for the container work.
+A proper `cmd` usually looks like /`[ root of <js_image_layer> ]`/`[ relative path to BUILD file from WORKSPACE or package_name() ]/[ name of <js_binary> ]`, 
```

Comment:
The `<` seems to be showing up as `&lt;` in the .md which doesn't seem correct? At least looks off in the github ui...

Reply:
removed those. IDK why they don't work